Abracadabra/AbracadabraAPI/AbracadabraAPI/Controllers/FlagsController.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using AbracadabraAPI.Authentication;
+using AbracadabraAPI.Data;
+using AbracadabraAPI.Models;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+
+namespace AbracadabraAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class FlagsController : ControllerBase
+    {
+        private readonly AbracadabraContext _context;
+        private readonly UserManager<ApplicationUser> userManager;

```

Comment:
_userManager is nicer if it is a readonly

Reply:
I agree we should keep it consistent, but the "_" is related to the variable being a member variable, not whether or not it's read only ;)